Cargo.toml

```diff
@@ -33,9 +33,9 @@ num-traits = "0.2.15"
 once_cell = "1"
 rand = "0.8.5"
 reqwest = { version = "0.11.13", features = ["json"] }
-schemars = "0.8"
+schemars = "0.8.12"
 serde = { version = "1", default-features = false, features = ["derive"] }
-serde-map-to-array = "1.1.0"
+serde-map-to-array = {version = "1.1.1", features = ["json-schema"]}
```

Comment:
I don't believe we should include the `json-schema` feature (see later comment).